Replicator/Replicator.cc

```diff
@@ -899,6 +899,7 @@ namespace litecore { namespace repl {
         _getCollectionsRequested = true;
         
         for (int i = 0; i < _collections.size(); i++) {
+            _sessionCollections.push_back(_collections[i].get());
```

Comment:
I forgot the case of using default collection only, which will call getRemoteCheckpoint(). I guess when calling getRemoteCheckpoint() first time, we need to populate sessionCollections as well?

Reply:
Similarly in handleGetCheckpoint(). Seem like we need a design about how to handle both collection-index and legacy BLIP messages.

Reply:
Right, this will also depend on your other comment whether we need to completely drop the collection index in this case or not.